docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+An incorrect position of a parameter resulted in the args passed as input instead of as args.
+
+
+### remove unsupported -showwindow switch from get-help [#4903](https://github.com/PowerShell/PowerShell/issues/4903)
+`-showwindow` relies on WPF, which is not supported on CoreCLR.
+
+
+### allow * to be used in registry path for remove-item [#4866](https://github.com/PowerShell/PowerShell/issues/4866)
+Using `-literalpath` with an asterisk and the filesystem provider will now return an error rather than returning quietly.
+
+TODO: Comibine with 5197?
+
+
+### fix set-service failing test [#4802](https://github.com/PowerShell/PowerShell/issues/4802)
+`New-Service` should throw on System `-StartupType`
+TODO : Looks like this a change in test cases, not sure what the documentation process should be
+
+### Enhance the -split operator with negative maximum token counts to split from the end [#4765](https://github.com/PowerShell/PowerShell/issues/4765)
+After the enhancement, only a `<Max-substrings>` value of `0` will be accepted as the explicit signal that all tokens should be returned, however many are found in the input string(s).
+Negative `<Max-substrings>` values will work analogously to the already supported positive values, except that: they return the specified number of strings from the end of the string(s) and all individually extracted tokens are returned in input order, and whatever unsplit part remains, if any, is returned as the first token.
+
+
+### Rename $IsOSX to $IsMacOS [#4700](https://github.com/PowerShell/PowerShell/issues/4700)
+The naming in PowerShell should be consistent with our naming andd conform to Apple's use of macOS instead of OSX, however, for readability and consistently we are staying with Pascal casing.
+
+
+### Prepare for BOM-less UTF-8 default character encoding with respect to $OutputEncoding and console code page  [#4681](https://github.com/PowerShell/PowerShell/issues/4681)
+Change `$OutputEncoding` to be utf8 without BOM rather than ASCII as BOM-less UTF-8 character encoding is the default for PowerShell Core.
+
+
+### Make error message consistent when invalid script is passed to -File, better error when passed ambiguous arg [#4573](https://github.com/PowerShell/PowerShell/issues/4573)
+Change the exit codes of pwsh.exe to align with Unix conventions
+
+
+### Removal of `LocalAccount` and cmdlets from  `Diagnostics` modules. [#4302](https://github.com/PowerShell/PowerShell/issues/4302) [#4303](https://github.com/PowerShell/PowerShell/issues/4303)
+Due to unsupported APIs, the `LocalAccounts` module and the `Counter` cmdlets in the `Diagnostics` module were removed until a better solution is found. 
+
+
+### Remove UTC and SQM telemetry code [#4190](https://github.com/PowerShell/PowerShell/issues/4190)
+TODO: Should this still be in the doc if the change is not visible to the user and is about telemetry collection?
```

Comment:
This shouldn't be in the list, it's not customer facing

Reply:
Agreed, removed